build.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
-VERSION="1.1.5"
+VERSION="1.2.0"
 MILESTONE="master"
-RPM_RELEASE="1.1.$MILESTONE.$(date -u +%Y%m%d%H%M%S)"
+RPM_RELEASE="1.2.$MILESTONE.$(date -u +%Y%m%d%H%M%S)"
```

Comment:
I would expect this to be 
RPM_RELEASE="1"

Reply:
Yes, that should be a stable version

Reply:
Done